lib/src/rules/avoid_init_to_null.dart

```diff
@@ -7,20 +7,20 @@ import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:linter/src/analyzer.dart';
 import 'package:linter/src/util/dart_type_utilities.dart';
 
-const desc = r"Don't explicitly initialize variables to null.";
+const _desc = r"Don't explicitly initialize variables to null.";
+
+const _details = r'''
 
-const details = r'''
 From [effective dart]
 (https://www.dartlang.org/effective-dart/usage/#dont-explicitly-initialize-variables-to-null):
 
 **DON'T** explicitly initialize variables to null.
 
 In Dart, a variable or field that is not explicitly initialized automatically
-gets initialized to null. This is reliably specified by the language. There's
-no concept of "uninitialized memory" in Dart. Adding `= null` is redundant and
+gets initialized to null.  This is reliably specified by the language.  There's
+no concept of "uninitialized memory" in Dart.  Adding `= null` is redundant and
```

Comment:
"= null" --> "== null"

Reply:
I agree.  I'll merge this.  @aemino (or anyone 😉 ): feel free to follow up with any fixes.  Tag and I'll be happy to review. 👍